internal/provider/provider.go

```diff
@@ -168,10 +168,13 @@ func getRetryConfig(d *schema.ResourceData) (kentikapi.RetryConfig, error) {
 		return kentikapi.RetryConfig{}, fmt.Errorf("parse %v duration: %v", maxDelayKey, err)
 	}
 
+	// KentikAPI returns http 500 when creating CloudExport with name that is already taken.
+	// This is non-retryable situation, so exclude http 500 from RetryableStatusCodes
```

Comment:
The fix is good. I'll file a bug against the back-end code to return a 400 error with informative error message in this case.